schedview/examples/versionsattime.py

```diff
@@ -0,0 +1,56 @@
+import argparse
+import datetime
+
+from astropy.time import Time
+
+import schedview.collect
+
+
+def versions_at_time(
+    iso_time: str | datetime.datetime,
```

Comment:
In astropy, this is called "isot" rather than "iso" isn't it? 
I wonder if keeping that convention would be useful, to avoid someone passing an iso format instead of isot format.